lib/ui/shared/smart_widgets/thread_card/widgets/thread_card_channel_main.dart

```diff
@@ -29,63 +27,25 @@ class ThreadChannelMain extends ViewModelWidget<ThreadCardViewModel> {
     return GestureDetector(
       onTap: () => viewModel.navigateToThread(userPost),
       child: Container(
-        padding: const EdgeInsets.symmetric(horizontal: 15),
+        padding: const EdgeInsets.symmetric(horizontal: 16.0),
         child: Row(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             InkWell(
               onTap: viewModel.viewProfile,
-              onLongPress: () => zuriChatBottomSheet(
```

Comment:
And where did this all zuriChatBottomSheet go to?

Reply:
That `onLongPress` callback passed to that `InkWell` is really superfluous and opposes the DRY principle. This is because an `onLongPress` that does basically the same thing is already being passed to the parent `InkWell` found on line 31 in [channel_chat.dart](https://github.com/ugoostanleyibe/zc_app/blob/dev/lib/ui/view/channel/channel_view/widgets/channel_chat.dart), which is the main list item that holds a user's post!